pyscitt/setup.py

```diff
@@ -25,7 +25,7 @@
     },
     python_requires=">=3.12",
     install_requires=[
-        "ccf==6.0.3",
```

Comment:
Same remark as for the test code, this is probably counter productive if there are fixes in the ccf ledger tooling